docs/personalization/how_it_works.md

```diff
@@ -9,7 +9,7 @@ The idea behind the Personalization service is built upon four main steps.
 1. A user visits a news service or an online store, navigates through it and leaves some footprints. The most popular footprints are selecting (CLICK) and purchasing products (BUY).
 2. This information is sent to the Personalization client every time these events happen.
 3. The Personalization server accumulates the tracked events, categorizes them and calculates recommendations for every product and every user in the shop.
-4. Recommendations can be fetched from the Personalization service and presented to users during their sessions.
+4. Recommendations can be fetched from the Personalization service and presented to users during their sessions or pushed by the service to a dedicated endpoint and then sent in an email.
```

Comment:
Should we mark here `then sent in an email`? It is customer responsibility what `dedicated endpoint` will do with those recommendations.